web/src/components/legend/Legend.tsx

```diff
@@ -0,0 +1,10 @@
+import type { ReactElement } from 'react';
+import Co2Legend from './Co2Legend';
+
+export default function Legend(): ReactElement {
```

Comment:
Can we call it something like this so it is clear that it's a "holder of legends"? :)
```suggestion
export default function LegendsContainer(): ReactElement {
```

Reply:
added